RunLog/RunLog/Sources/Presentation/Log/View/EditLog/EditLogInfoView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  EditLogInfoView.swift
+//  RunLog
+//
+//  Created by 김도연 on 3/18/25.
+//
+
+import UIKit
+import SnapKit
+import Then
+
+final class EditLogInfoView: UIView {
+    
+    // MARK: - Properties
+    private let placeHolderString = "기록 이름을 필수로 지정해 주세요"
+    
+    private lazy var titleLabel = UILabel().then {
+        $0.numberOfLines = 1
+        $0.textAlignment = .left
+        $0.attributedText = .RLAttributedString(text: "제목", font: .Heading2)
+    }
```

Comment:
```suggestion
    private lazy var titleLabel = UILabel().then {
        $0.numberOfLines = 1
        $0.attributedText = .RLAttributedString(text: "제목", font: .Heading2)
    }
```
기본값으로 .left alignment가 들어가있습니다!! 추가로 left가 아닌 align 지정해주실거면
.RLAttributedString(text: "제목", font: .Heading2, align: .center) 요렇게 넣어주시면됩니다!
(쓰면서 드는 생각이 이 부분 전에 얘기를 했었나요....? 먼가 데자뷰가...)

Reply:
아마 예전 코드 그대로 가져와서 그럴거에요 ㅎㅎ 한번에 고치겠습니닷~